src/tests/specs/workitem-naughty-string.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { browser } from 'protractor';
+import { PlannerPage } from '../page_objects/planner';
+import * as support from '../support';
+import { SidePanel } from './../ui/planner/sidepanel';
+
+
+describe('Work Item list: ', () => {
+  let planner: PlannerPage;
+  let plannerAgile: PlannerPage;
+  let c = new support.Constants();
+
+  beforeAll(async () => {
+    await support.desktopTestSetup();
+    planner = new PlannerPage(browser.baseUrl);
+    await planner.openInBrowser();
+    let url = await browser.getCurrentUrl();
+    let urlPathName: any = await browser.executeScript('return document.location.pathname');
+    let URL = url.replace(urlPathName, '/' + process.env.USER_NAME + '/' + process.env.SPACE_NAME_SCRUM + '/plan');
+    plannerAgile = new PlannerPage(URL);
+    await browser.get(URL);
+    await plannerAgile.waitUntilUrlContains('typegroup');
+  });
+
+  beforeEach(async () => {
+    await plannerAgile.workItemList.overlay.untilHidden();
+    await plannerAgile.sidePanel.workItemsGroupAgile.clickWhenReady();
+  });
+
+  it('Should Quick create UnicodeSymbols workitems', async () => {
+    let newWorkItem = {
+      title: 'Ω≈ç√∫˜µ≤≥÷åß∂ƒ©˙∆˚¬…æœ∑´®†¥¨ˆøπ“‘¡™£¢∞§¶•ªº–≠¸˛Ç◊ı˜Â¯˘¿ÅÍÎÏ˝ÓÔÒÚÆ☃Œ„´‰ˇÁ¨ˆØ∏”’`⁄€‹›ﬁﬂ‡°·‚—±⅛⅜⅝⅞ЁЂЃЄЅІЇЈЉЊЋЌЍЎЏАБВГДЕЖЗИЙКЛМНОПРСТУФХЦЧШЩЪЫЬЭЮЯабвгдежзийклмнопрстуфхцчшщъыьэюя٠١٢٣٤٥٦٧٨٩'
+    };
+    await plannerAgile.createWorkItem(newWorkItem);
```

Comment:
Shouldn't we create workitem in Agile and SDD since we have `planner` and `plannerAgile`?
I think we don't need both `planner` and `plannerAgile`.

Reply:
Yes, @jarifibrahim planner was not used anywhere, now URL is updated to directly use Agile Space